pkg/web/apps/apps_psql_repository.go

```diff
@@ -0,0 +1,35 @@
+package apps
+
+import (
+	"github.com/aerogear/mobile-security-service/pkg/models"
+	"github.com/labstack/echo"
+)
+
+type (
+	// PostgreSQLRepository interface defines the methods to be implemented
+	PostgreSQLRepository interface {
+		GetApps(c echo.Context) (*[]models.App, error)
+	}
+
+	appsPostgreSQLRepository struct {
+		// TODO: Add Db connection
+	}
+)
+
+// NewPostgreSQLRepository creates a new instance of appsPostgreSQLRepository
+func NewPostgreSQLRepository() PostgreSQLRepository {
+	return &appsPostgreSQLRepository{}
+}
+
+// GetApps retrieves all apps from the database
+func (a *appsPostgreSQLRepository) GetApps(c echo.Context) (*[]models.App, error) {
+	app1 := models.App{
+		ID:      1,
+		AppID:   "com.aerogear.app1",
+		AppName: "app1",
+	}
```

Comment:
We will need to go under the assumption that it is possible to retrieve the other details needed(number of versions, clients and appLaunches) from the same db query. Until that assumption changes, this should be reflected in the code in my opinion.